lib/badge/runner.rb

```diff
@@ -121,7 +125,8 @@ def load_shield(shield_string)
       UI.verbose "Trying to load image from shield.io. Timeout: #{Badge.shield_io_timeout}s".blue
       UI.verbose "URL: #{url}".blue
 
-      Curl::Easy.download(url, file_name)
+      #Curl::Easy.download(url, file_name)
+      MiniMagick::Image.open(file_name)
```

Comment:
Please uncomment line with Curl and leave `MiniMagick::Image.open(file_name)` as it is. I commented this line to test purposes and forget to uncomment it :/